MembraneRTC/Package.swift

```diff
@@ -16,7 +16,7 @@ let package = Package(
         )
     ],
     dependencies: [
-        .package(name: "WebRTC", url: "https://github.com/webrtc-sdk/Specs.git", .exact("104.5112.14")),
+        .package(name: "WebRTC", url: "https://github.com/webrtc-sdk/Specs.git", .exact("104.5112.15")),
```

Comment:
please bump the version also in `.podspec` file (see here: https://github.com/jellyfish-dev/membrane-webrtc-ios/commit/b8749af1218b8be754e65781035a9d66b0ea46cf)

The reason why we have the version specified in two places is that swift has two package managers (swift-package and pods) and we use both of them. Pods are needed for react-native and swift-package was there before and I've never really thought about removing it, someone may use it in projects that use swift-package and it's convenient for ios developing

Reply:
I added dependency in the `.podspec`. However I find it super counterintuitive to use simultaneously two package managers. And mby we should think about it if we were to have monorepo for sdks